app/models/filters/filter_base.rb

```diff
@@ -1392,6 +1393,12 @@ def to_comparison
         prior_end = end_date - 1.years
         prior_start = start_date - 1.years
         [prior_start, prior_end]
+      when :prior_fiscal_year
+        # find the 9/30 that precedes the end date
+        prior_end = Date.new(end_date.year, 9, 30)
+        prior_end -= 1.years if prior_end > end_date
```

Comment:
I think this should be
```suggestion
        prior_end -= 1.years if prior_end >= end_date
```
I'm thinking that if I select the current fiscal year, then `prior_end == end_date` and we end up with both periods being the same?

Reply:
Agreed.  Thanks!